roachtest: gopg failed

roachtest.gopg [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/20904602?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/20904602?buildTab=artifacts#/gopg) on release-25.4 @ [3453896dbd480402480bb80a3d89653a3116ed9c](https://github.com/cockroachdb/cockroach/commits/3453896dbd480402480bb80a3d89653a3116ed9c):


```
(orm_helpers.go:225).summarizeFailed: 
Tests run on Cockroach v25.4.3-dev-3453896dbd480402480bb80a3d89653a3116ed9c
Tests run against gopg v10.9.0
195 Total Tests Run
178 tests passed
17 tests failed
0 tests skipped
7 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: v10.TestGocheck - unknown (unexpected)
For a full summary look at the gopg artifacts 
An updated blocklist (gopgBlockList) is available in the artifacts' gopg log
(cluster.go:2501).Run: context canceled
test artifacts and logs in: /artifacts/gopg/run_1
```

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=gce</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=4</code>
 - <code>encrypted=false</code>
 - <code>metamorphicBufferedSender=true</code>
 - <code>metamorphicLeases=epoch</code>
 - <code>runtimeAssertionsBuild=false</code>
 - <code>ssd=0</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-20904602/gopg/1766563158211/1766563485524)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #159481 roachtest: gopg failed [B-runtime-assertions-enabled C-test-failure O-roachtest O-robot P-3 T-sql-foundations branch-release-25.4.2-rc]
- #158483 roachtest: gopg failed [B-runtime-assertions-enabled C-test-failure O-roachtest O-robot P-3 T-sql-foundations branch-release-25.2.10-rc]
</p>
</details>
/cc @cockroachdb/sql-foundations
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*gopg.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-58162

--- a/pkg/cmd/roachtest/tests/gopg_blocklist.go
+++ b/pkg/cmd/roachtest/tests/gopg_blocklist.go
@@ -53,6 +53,8 @@ var gopgIgnoreList = blocklist{
 	`pg | ORM | relation with no results does not panic`:            "unknown",
 	// This test flakes sometimes because of connection reuse.
 	`v10.TestColumnReuse`: "unknown",
+	// This test is flaky.
+	`v10.TestGocheck`: "unknown",
 	// This test is flaky sometimes due to the use of temp tables.
 	`pg | soft delete with int column nil model ForceDelete | deletes the model`: "unknown",
 	// This test is sometimes flaky due to a small delta in the expected soft

